browser/ui/webui/welcome_page/welcome_dom_handler.cc

```diff
@@ -64,6 +65,13 @@ bool IsChromeDev(const std::u16string& browser_name) {
          browser_name == kChromeDevLinuxBrowserName;
 }
 
+bool ShouldRedirectToGettingStartedPage() {
+  return base::FeatureList::IsEnabled(features::kBraveDayZeroExperiment) &&
```

Comment:
perhaps we should change this to:

```
  return (base::FeatureList::IsEnabled(features::kBraveDayZeroExperiment) &&
         features::kBraveDayZeroExperimentVariant.Get() == "c") ||
         base::FeatureList::IsEnabled(
             brave_education::features::kShowGettingStartedPage);
```

see https://bravesoftware.slack.com/archives/C05K43DL8JW/p1743625312678449?thread_ts=1739223993.051799&cid=C05K43DL8JW

Reply:
Fixed